src/pages/signin/SignInPageLayout/index.js

```diff
@@ -13,11 +13,19 @@ const SignInPageLayout = props => (
             <SignInPageLayoutWide
                 welcomeText={props.welcomeText}
                 isMediumScreenWidth={props.isMediumScreenWidth}
+                shouldShowWelcomeText={props.shouldShowWelcomeText}
```

Comment:
Can you please add `shouldShowWelcomeText` to `propTypes`? Maybe even `welcomeText` too since it's missing